storages/utils.py

```diff
@@ -1,9 +1,11 @@
 import posixpath
 
 from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
 from django.utils.encoding import force_text
 
 
+
```

Comment:
This extra new line is causing flake8 to fail. If you remove it Travis should be green again.